sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/bridge/AnalysisProcessor.java

```diff
@@ -70,6 +71,7 @@ public class AnalysisProcessor {
   private ContextUtils contextUtils;
   private InputFile file;
   private JsTsChecks checks;
+  HashSet<String> parsingErrors;
```

Comment:
You would need to rename its usages below as well.

```suggestion
  HashSet<String> parsingErrorFiles;
```

Reply:
or just store the counter integer...

Reply:
ah, makes sense. Could you mention it somewhere?